src/app/shared/experience-preview/experience-preview.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, OnInit, Output} from '@angular/core';
+import {IExperience} from '../../core/user/user.types';
+import {ProfileService} from "../../service/profile.service";
+
+@Component({
+    selector: 'app-experience-preview',
+    templateUrl: './experience-preview.component.html',
+    styleUrls: ['./experience-preview.component.scss']
+})
+export class ExperiencePreviewComponent implements OnInit {
+
+    @Input() experienceList: IExperience[];
+    @Input() onDelete: (id: number) => void;
```

Comment:
@Input() onDelete: (id: number) => void;
what is this representing ? This should never be a function, If you want to pass an event you can use @Output with event emitters